test/output-tests/syn_misc.expected

```diff
@@ -223,7 +223,7 @@ syn_misc.at-1706-prog.cob:12.24:
 ----                           ^
   13              ACCEPT X WITH NO UPDATE  END-ACCEPT.
   14              ACCEPT X WITH NO DEFAULT END-ACCEPT.
->> Hint: Missing UPDATE
+>> Hint: Missing AUTO
```

Comment:
Are we still missing the `WITH DEFAULT`…?

Reply:
Yeah, this one is neither COBOL 2023 nor Microfocus.

Reply:
Seems like it's an ACUCOBOL stuff. Though ACUCOBOL does not have the negative forms (`NO DEFAULT` , `NO UODATE`)

Reply:
Can be left for later. Can you try to see if the `[@symbol …]` suggestion fixes the hint?

Reply:
That results in this diff :
```
->> Hint: Missing AUTO
+>> Hint: Missing rnell_rev_loc_accept_with_clause__
```
Pretty weird, ain't it ?

Reply:
Hum… maybe the `rnell` is messing things up. What do you get with `nell` that you used originally instead.

Reply:
With `nell` i just get `>> Hint: Missing AUTO`.

Reply:
Done. Works better (still need to use `nell` instead of `rnell` BTW).

Reply:
Ok thanks for the notice. Maybe `rnell` always messes up hints (I need to check that) (ideally we'd need the hint/symbol as a parameter to `rnell`… but I guess that's not covered by menhir ;-)